src/rustc/fortanix-sgx-abi_shim/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "fortanix-sgx-abi"
```

Comment:
Could this be published to crates.io for future use in https://github.com/rust-lang/rust/pull/56092?

Reply:
https://crates.io/crates/fortanix-sgx-abi